CU-1kjd67v-Connect-client-Update-verifyJWT-to-take-an-optional-audience-parameter (#453)
fewlinesco/connect-js: client/README.md

### verifyJWT

```typescript
async verifyJWT<T = unknown>(accessToken: string, algo: string, audience?: string): Promise<T> {}
```

> Used to verify the JWS (i.e. `access_token`). It provides a series of checks, like audiences, algorithm or public key. The audience parameter is optional and will have a default value of the config audience.

Reviewer: ```suggestion
Used to verify the JWS (i.e. `access_token`). It provides a series of checks, like audiences, algorithm or public key. The audience parameter is optional and will default to the value you provided when creating the client.
```
